vignettes/tables.Rmd

```diff
@@ -86,7 +86,7 @@ moreGenes<-data.frame(
 	Strand=c('+', '+'),
 	TranscriptionFactor=c(F,F))
 moreGenesFile<-tempfile()
-write.csv(moreGenes, file=moreGenesFile, row.names=FALSE)
+saveToCsv(moreGenes, moreGenesFile)
```

Comment:
In one of the examples in the tables vignette please show how to enter a missing value (i.e., NA)